experimental/traffic-portal/src/app/core/cache-groups/cache-group-table/cache-group-table.component.ts

```diff
@@ -286,6 +286,15 @@ export class CacheGroupTableComponent implements OnInit {
 			case "dequeue":
 				this.queueUpdates(Array.isArray(a.data) ? a.data : [a.data], false);
 				break;
+			case "asns":
+				let cg: ResponseCacheGroup;
+				if (Array.isArray(a.data)) {
+					cg = a.data[0];
+				} else {
+					cg = a.data;
+				}
+				this.router.navigate(["/core/asns"], {queryParams: {cachegroup: cg.name}});
+				break;
```

Comment:
This is only doing navigation, so it should be a link, not a button handling a click event

Reply:
But it is a button on context-menu, which one has to click to navigate.

Reply:
Navigation should be handled by anchor tags. It's more accessible, provides more out-of-the-box functionality, and more intuitive for users.